src/conductor.ts

```diff
@@ -81,7 +81,8 @@ export class Conductor {
       agent_key: agentPubKey,
       dnas: dnaPaths.map((dnaPath, index) => ({
         path: dnaPath,
-        nick: `${index}${dnaPath}-${uuidGen()}`
+        nick: `${index}${dnaPath}-${uuidGen()}`,
```

Comment:
Should we update `uuidGen()` to `this._player.scenarioUUID`?

Reply:
I think it should different across each call to installHapp that would keep it constant.